.github/workflows/nodejs_build.yml

```diff
@@ -0,0 +1,24 @@
+name: Node.js CI
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
Let's use only `20.x` for now

Reply:
Modifying it now

Reply:
The change was applied to the node version